### PR TITLE
Added labels and annotations to Helm Chart

### PR DESCRIPTION
**Pull request by SylwiaBrant**
Added configuration option for the user to specify annotations and labels for Jenkins CR and pod via helm. Updated installation guide with information about this feature.
#432

### CODE REVIEW

File: website/content/en/docs/Installation/_index.md [hunk 1]
```diff
@@ -56,5 +56,11 @@ To install, you need only to type these commands:
...
 ```bash
 $ helm repo add jenkins https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/chart
-$ helm install jenkins/jenkins-operator
+$ helm install <name> jenkins/jenkins-operator
+```
+
+To add custom labels and annotations, you can use `values.yaml` file or pass them into `helm install` command, e.g.:
+
+```bash
+$ helm install <name> jenkins/jenkins-operator --set jenkins.labels.LabelKey=LabelValue,jenkins.annotations.AnnotationKey=AnnotationValue
```
Comment (jakalkhalili) [inline website/content/en/docs/Installation/_index.md:65]: `helm install <name> jenkins/jenkins-operator --set jenkins.labels.LabelKey=LabelValue,jenkins.annotations.AnnotationKey=AnnotationValue` -> `helm install <name> jenkins/jenkins-operator`. 

Add separated section with info about `values.yaml` customization possibility.

**Review by jakalkhalili — CHANGES_REQUESTED** [review]

File: website/content/en/docs/Installation/_index.md (hunk 1 above)
Comment (SylwiaBrant) [reply website/content/en/docs/Installation/_index.md:65]: Since I'm already changing docs later (recommiting to fix_docs pull request) I guess I can add it then to not generate docs twice? We could do separate page about customizing with helm and leave a link there or add a 'helm section' in existing customization page.

File: chart/jenkins-operator/templates/jenkins.yaml [hunk 2]
```diff
@@ -7,6 +7,9 @@ metadata:
   {{- with .Values.jenkins.labels }}
   labels: {{ toYaml . | nindent 4 }}
   {{- end }}
+  {{- with .Values.jenkins.annotations }}
+  annotations: {{ toYaml . | nindent 4 }}
+  {{- end }}
```
Comment (tomaszsek) [inline chart/jenkins-operator/templates/jenkins.yaml:12]: The annotations should be configurable only for Jenkins pod, not Jenkins instance. Please revert this change.

File: chart/jenkins-operator/templates/jenkins.yaml [hunk 3]
```diff
@@ -133,4 +139,4 @@ spec:
   {{- with .Values.jenkins.seedJobs }}
   seedJobs: {{- toYaml . | nindent 4 }}
   {{- end }}
-{{- end }}
+  {{- end }}
```
Comment (tomaszsek) [inline chart/jenkins-operator/templates/jenkins.yaml:142]: Please revert this line(formatting).

**Review by tomaszsek — CHANGES_REQUESTED** [review]

**Review by jakalkhalili — APPROVED** [review]

**Review by tomaszsek — APPROVED** [review]
